src/GraphODataTemplateWriter/CodeHelpers/Android/TypeHelperAndroid.cs

```diff
@@ -37,6 +37,13 @@ public static string GetTypeString(this OdcmType @type)
                     return "com.microsoft.graph.model.DateOnly";
                 case "Binary":
                     return "byte[]";
+                case "Stream":
+                    return "java.io.InputStream";
+                case "Duration":
+                    return "javax.xml.datatype.Duration";
+                case "TimeOfDay":
+                case "Json":
```

Comment:
This is true, although [my implementation](https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/blob/master/src/GraphODataTemplateWriter/CodeHelpers/Android/TypeHelperAndroid.cs#L38-L39) differs and relies upon the [gson dependency](https://github.com/microsoftgraph/msgraph-sdk-android/blob/master/graphsdk/build.gradle#L62). @mikemcl-msft With justification, I could be convinced that there may be a reason not lock this implementation to this dependency and would happily accept/make such a change